Documentation/ApiOverview/SiteHandling/SiteSettings.rst

```diff
@@ -34,8 +34,24 @@ For instance, settings can be used in custom frontend code to deliver features
 which might vary per site for extensions. An example may be to configure
 storage page IDs.
 
-The settings are defined in the :file:`config/sites/<my_site>/settings.yaml <set-settings-yaml>`
-file.
+All changed values in the site settings editor are stored in the `settings.yaml`
+file of the `config/sites/<my_site>/` directory. If this file does not exists,
+it will be created.
+
+..  note::
+    You *can* make use of `imports` here, but with next storing process of
+    site settings editor all your `imports` are gone.
+
+To store values independant from site settings editor, use the `settings.yaml`
+file in folder `Configuration/Sets/<my_site>/` of your site package. The file
+itself will kept untouched, but the values will be overwritten by values from the
+`settings.yaml` in the `config/sites/<my_site>/`.
+
+..  note::
+    As this file is independant from external modification, you can make use
+    of `imports` here, if needed. Absolute paths, relative paths and of cause
```

Comment:
We should avoid "of course", as this might not be understandable by the reader why this is "of course".